Month1/Week2/Section2/Month1-Week2-Section2/Spotify/Song.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include <string>
+#include <iostream>
+#include <string>
+class FSong
+{
+private:
+    int MinutesDuration;
```

Comment:
Please follow the exercise's instructions closely. You were supposed to just have one variable for the duration which holds the total seconds of the song. It is important to differentiate the way we want to store our data and the way we want to display it. Of course, to display the duration, the format `minutes:seconds`, but it is far less complicated to store the data as just one variable. That way there aren't different values storing the same data.

Reply:
Changed